js/msp/MSPHelper.js

```diff
@@ -617,6 +617,20 @@ var mspHelper = (function (gui) {
                 offset += 2;
                 FAILSAFE_CONFIG.failsafe_procedure = data.getUint8(offset);
                 offset++;
+                FAILSAFE_CONFIG.failsafe_recovery_delay = data.getUint8(offset);
```

Comment:
Those settings were introduced only around February, older versions of firmware will not send them. We should find a release that intrudced them and use this data segment only is firmware version allows for it